### PR TITLE
Add openstack DNS operations

### PR DESCRIPTION
**Pull request by fabioDMFerreira**
#718

### CODE REVIEW

File: lepton/openstack_dns.go [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+	allZones, err := zones.ExtractZones(allPages)
+	if err != nil {
+		return "", err
+	}
+
+	if len(allZones) == 0 {
+		createOpts := zones.CreateOpts{
+			Name:  dnsName + ".",
+			Email: "martinhoferreira10@gmail.com",
```
Comment (eyberg) [inline lepton/openstack_dns.go:37]: ? is this required or ?

**Review by eyberg — APPROVED** [review]
just need to fix the email

File: lepton/openstack_dns.go (hunk 1 above)
Comment (fabioDMFerreira) [reply lepton/openstack_dns.go:37]: I added my personal email by accident. Without email parameter, it isn't possible to create a zone in openstack.

Comment (eyberg) [reply lepton/openstack_dns.go]: that seems weird? if you need something you can just use the example.com from http://www.faqs.org/rfcs/rfc2606.html

Comment (fabioDMFerreira) [reply lepton/openstack_dns.go]: I added a default email with format `admin@<domain name`. I guess it makes sense the user to add a command flag that sets his email.

**Pull request merged**